src/classes/Options.ts

```diff
@@ -391,6 +392,7 @@ export const DEFAULTS = {
         avatarURL: '',
         embedColor: '9171753',
         tradeSummary: {
+            declinedTrade:{enable:false},
```

Comment:
well, I didn't mean here.
It's here:
https://github.com/TF2Autobot/tf2autobot/blob/f719fefbebd6076c20f4eb92e202cb8a8b4c79ae/src/classes/Options.ts#L117-L118

Reply:
Well mb ctrl+f'd tradeSummary and pasted it there without actually looking where I was fixing it rn.

Reply:
Haha understand 😅

Reply:
I should probably stop coding late especially like now when is literally 6 am